content/articles/implementation-of-ci-and-cd-in-.net-application-using-shell-executor-on-gitlab/index.md

```diff
@@ -0,0 +1,136 @@
+To install and run GitLab Runner on Windows, you'll need Git and the User password. For more information, see the Git download link in the prerequisites section. 
+
+`step-by-step guide`
+1.  Make a folder on your computer first. For example `H:\Runners`
+
+2. To install the GitLab runner, first, download the binaries for either the Windows 64-bit or 32-bit operating systems and place them in the folder you created. Please note that the following assumes that you've changed GitLab-runner.exe to GitLab-runner (optional).
+
+> The binary down depends on your computer specifications of the operating system.
+
+3. Make sure that the GitLab Runner directory and executable have `Write rights` restricted. Otherwise, regular users can replace the executable and run any code with elevated privileges by substituting their own.
+
+4. Open a command prompt as an administrator.
+
+5. Make registration for the runner. This can be achieved by the process of connecting a runner to one or more GitLab instances.
+Registering multiple runners is as simple as repeating the register command on the same host system.
+
+The following is the process of registering a runner.
+
+- Run the command `./gitlab-runner.Exe register` to register a runner
+- The URL of your GitLab instance must be entered (also known as the GitLab-ci coordinator URL).
+- To register a runner, enter the token you received.
+- Name and describe the runner. This value can be changed later through the GitLab user interface.
+- Separate the tags linked with the runner using commas. This value can be changed later through the GitLab user interface.
+- Provide the executor for the runner. Enter docker for the majority of usage cases.
+- For projects that don't have a default image defined, you'll be requested to select one if you selected docker as your executor. GitLab-ci.yml.
+
+6. Install GitLab Runner as a service and start it. You can either use the built-in System Account using the command `.\gitlab-runner.Exe installs` to install and `.\gitlab-runner.exe start` to start the runner (which is preferred) or a user account to execute the service. Use the commands `.\gitlab-runner.exe install --user "input your user name" --password "Input your password"` to install and 
+`.\gitlab-runner.exe start` to start the runners.
+
+### Configuring the runners
+GitLab Runner is a build instance that can be used to perform jobs across several machines and transmit the results to GitLab. It can be installed on different users' computers, servers, and local devices. Once the runner has been installed, you can choose to make it shared or specific.
+> You can use either specific or shared runners to service your jobs.
+
+**Shared Runners**- These runners are handy for jobs that have similar requirements across numerous projects. Instead of using several runners for multiple projects, you may handle multiple projects with a single or small number of Runners that are easy to maintain and update

```

Comment:
```suggestion
Shared Runners- These runners are handy for jobs that have similar requirements across many projects. Instead of using several runners for many projects, you may handle them with a single or small number of runners that are easy to maintain and update.
```